comms/proton_queue_adaptor.py

```diff
@@ -65,6 +69,13 @@ async def send_async(self, message: dict, properties: Dict[str, Any] = None) ->
         except MaxRetriesExceeded as e:
             raise MessageSendingError() from e
 
+    def wait_for_messages(self):
```

Comment:
declared interface is async but implementation is not

Reply:
corrected